src/sentry/api/endpoints/organization_projects.py

```diff
@@ -97,6 +97,8 @@ def get(self, request, organization):
                 elif key == "!team":
                     team_list = list(Team.objects.filter(slug__in=value))
                     queryset = queryset.exclude(teams__in=team_list)
+                elif key == "is_member":
```

Comment:
I assume you'll only pass this for non-superusers?

Reply:
yup that's the plan for now